### ISSUE TITLE
False positive in three.min.js

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#552]
```
$ ./build-profile/quick-lint-js /Users/strager/Downloads/three.min.js
/Users/strager/Downloads/three.min.js:3135:2649: error: invalid expression left of assignment [E0020]
/Users/strager/Downloads/three.min.js:3135:20040: error: invalid expression left of assignment [E0020]
/Users/strager/Downloads/three.min.js:3135:209546: error: invalid expression left of assignment [E0020]
/Users/strager/Downloads/three.min.js:3135:221514: error: invalid expression left of assignment [E0020]
```

**Comment by strager** [comment]
Here's the offending expression for the first diagnostic:
```js
Y!==void 0?C=Y:C=_.isPointLight===!0?l:a,s.localClippingEnabled&&E.clipShadows===!0&&E.clippingPlanes.length!==0||E.displacementMap&&E.displacementScale!==0||E.alphaMap&&E.alphaTest>0
```

**Comment by strager** [comment]
Fixed in commit 6f3e99ecfc846a8e5d03bc89e49e197eca80010b.
